assert/assertions.go

```diff
@@ -1250,19 +1254,19 @@ func InDeltaMapValues(t TestingT, expected, actual interface{}, delta float64, m
 
 func calcRelativeError(expected, actual interface{}) (float64, error) {
 	af, aok := toFloat(expected)
-	if !aok {
-		return 0, fmt.Errorf("expected value %q cannot be converted to float", expected)
+	bf, bok := toFloat(actual)
```

Comment:
I'm concerned that this might break some expected behaviour. I'm fine to let it slide for now as master but will tag it for version 1.8.0